apps/emqx/src/emqx_router.erl

```diff
@@ -784,23 +789,27 @@ mk_conflict_resolution_action(State) ->
     NodesV1 = [Node || {Node, v1, _} <- State],
     NodesUnknown = [Node || {Node, unknown, _} <- State],
     Format =
-        "Following EMQX nodes are running with conflicting schema:"
-        "\n ~p"
-        "Please take the following steps to resolve the conflict:"
-        "\n 1. Stop listeners on those nodes: `$ emqx eval 'emqx_listener:stop()'`"
-        "\n 2. Wait until they are safe to restart."
-        "\n This could take some time, depending on the number of clients and their subscriptions."
-        "\n The following conditions should be both true for each of the nodes in order to proceed:"
-        "\n   * `$ emqx eval 'ets:info(emqx_subscriber, size)'` prints `0`."
-        "\n   * `$ emqx ctl topics list` prints `No topics.`"
-        "\n 3. Upgrade the nodes to version ~s."
-        "\n 4. Restart the nodes.",
+        "There are two ways to resolve the conflict:"
+        "\n"
+        "\nA: Full cluster restart: stop ALL running nodes one by one "
+        "and restart them in the reversed order."
```

Comment:
Q: If this is full restart anyway, why the order is important?

Reply:
this is to avoid mnesia wait for table.
if node1 is stopped the first, and try to start it first, it may wait for other nodes to come up,
it's usually ok if all nodes are brought up around the same time, but could be confusing if one wants to start the node, then wait for status before proceeding to the next.